src/AutoComplete/AutoComplete.js

```diff
@@ -500,20 +510,26 @@ class AutoComplete extends Component {
     this.requestsList = requestsList;
 
     const menu = open && requestsList.length > 0 && (
-      <Menu
-        ref="menu"
-        autoWidth={false}
-        disableAutoFocus={focusTextField}
-        onEscKeyDown={this.handleEscKeyDown}
-        initiallyKeyboardFocused={true}
-        onItemTouchTap={this.handleItemTouchTap}
-        onMouseDown={this.handleMouseDown}
-        style={Object.assign(styles.menu, menuStyle)}
-        listStyle={Object.assign(styles.list, listStyle)}
-        {...menuProps}
+      <div
+        onMouseEnter={this.handleMouseEnter}
+        onMouseLeave={this.handleMouseLeave}
+        onBlur={this.handleBlur}
```

Comment:
Yeah, I tried that but it didn't work. It seems the scrollbar is applied directly to the `Menu` root element and we need a wrapper to capture clicks.